### PR TITLE
ch: Added set up the project landing page and user login and signup form

### LINKED ISSUE #1: Project setup and configuration.
Set up the project and install necessary dependencies that are required to run the project.

### PR DESCRIPTION
**Pull request by amosmoyo**
…ome, footer and login forms

## Front-end setup and configuration
I added bootstrap@4.4.1, jquery@3.4.1. I deed basic setup of the navbar, homepage, footer and the login pages.

## Checklist:
null

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
-barebones setup for JavaScript / Node projects replace this with a one-liner phrase describing this project or app
+# IamNotCrazyJustSick
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/6e7b0d2636d84f5787f6cd787fb32b67)](https://app.codacy.com/gh/BuildForSDG/team-132-frontend?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/team-132-frontend&utm_campaign=Badge_Grade_Settings)
-[![Codacy Badge](https://img.shields.io/badge/Code%20Quality-D-red)](https://img.shields.io/badge/Code%20Quality-D-red)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/511596ade6fd452697016a287bb9d1e4)](https://app.codacy.com/gh/BuildForSDG/Team-220-Frontend?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/Team-220-Frontend&utm_campaign=Badge_Grade_Settings)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/292178a7-2f85-4b07-a740-b1d0d27bc026/deploy-status)](https://app.netlify.com/sites/iamnotcrazyjustsick/deploys)
```
Comment (francis-mwas) [inline README.md]: have you hosted it on netlify

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { TestBed, async } from '@angular/core/testing';
```
Comment (francis-mwas) [inline src/app/app.component.spec.ts:1]: Put all your tests on tests folder.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
-barebones setup for JavaScript / Node projects replace this with a one-liner phrase describing this project or app
+# IamNotCrazyJustSick
```
Comment (cliffgor) [inline README.md]: Amos, please update the Readme File with your project details

File: README.md (hunk 1 above)
Comment (cliffgor) [inline README.md]: Update these Details Amos

**Review by didasmbalanya — COMMENTED** [review]
NIce work however how can I test manually?

**Comment by amosmoyo** [comment]
hey buddy, run "npm serve -o"

File: README.md (hunk 1 above)
Comment (amosmoyo) [reply README.md]: Have updated

**Review by francis-mwas — APPROVED** [review]
Great job.

**Pull request merged**